lib/__tests__/standalone-fix.test.js

```diff
@@ -246,7 +246,8 @@ it('one rule being disabled', async () => {
 		fix: true,
 	});
 
-	expect(JSON.parse(result.output)[0].errored).toBe(true);
+	expect(result.results[0].errored).toBe(true);
+	expect(result.output).toBe('\n		a {\n			color: red;\n		}');
```

Comment:
[suggest] What about making the code a bit smaller as follows?

```suggestion
	expect(result.output).toBe(code);
```

Reply:
sure, thanks for the suggestion.